src/services/CommunityService.ts

```diff
@@ -30,8 +29,14 @@ const request = async (): Promise<string> => {
         feedUrl = '/nemflash';
     }
     // execute request
-    const response = await axios.get(feedUrl, { params: {} });
-    return response.data;
+    const response = await fetch(feedUrl, {
+        method: 'GET',
+    })
+        .then((response) => response.text())
+        .then((str) => {
+            return str;
```

Comment:
Doesn't do anything right?

Reply:
Yes, and it was here because I was trying to create an XML document before creating a string out of it. Just updated it